mrt-services/docker-compose.yml

```diff
@@ -116,6 +116,23 @@ services:
     ports:
     - published: 3306
       target: 3306
+  ldap:
+    container_name: openDJ
+    image: cdluc3/mrt-opendj
+    build:
+      context: ldap
+      dockerfile: Dockerfile
+    networks:
+      merrittnet:
+    ports:
```

Comment:
Will we want to expose any of these ports for user browsing?  If so, our dev server will make 8080-8099 available.